boa/src/builtins/function/mod.rs

```diff
@@ -580,7 +582,16 @@ impl BuiltInFunctionObject {
                     .into())
                 }
             }
-
+            #[cfg(feature = "vm")]
+            (Function::VmOrdinary { .. }, Some(name)) if name.is_empty() => {
+                Ok("[Function (anonymous)]".into())
+            }
+            #[cfg(feature = "vm")]
+            (Function::VmOrdinary { .. }, Some(name)) => {
+                Ok(format!("[Function: {}]", &name).into())
+            }
+            #[cfg(feature = "vm")]
+            (Function::VmOrdinary { .. }, None) => Ok("[Function (anonymous)]".into()),
```

Comment:
Are these strings defined in the spec? If so, could we add a link to the definition?

Reply:
This is the `Function::toString` method. Most of it is implementation defined according to the spec: https://tc39.es/ecma262/#sec-function.prototype.tostring